docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+
+We open a second terminal window, to which we will refer using the name `terminal_B`.
+We use `terminal_B` for all the commands dealing with Pukki.
+
+Having a working command-line interface for Pukki is a prerequisite for continuing with the tutorial.
+If we haven't set it up before, we follow the [instructions on how to install and configure Pukki command-line interface](../../dbaas/cli.md#getting-started).
+
+We can test the correct functioning of the command-line interface by simply listing the available types of database.
+An example of the command and its expected output follows:
+```
+$ openstack datastore list
++--------------------------------------+------------+
+| ID                                   | Name       |
++--------------------------------------+------------+
+| 71920375-6967-466e-b955-8ee8629312b7 | postgresql |
+| 1a8efda2-7bb7-4c52-9eab-e251fd18323c | mariadb    |
++--------------------------------------+------------+
+```
+
+We now create the database that we use to log the actions of the pipeline by issuing the following command:
+```
+$ openstack database instance create pipeline_db_instance \
+--flavor standard.small \
+--databases pipeline_db \
+--users db_admin:db_password \
+--datastore postgresql \
+--is-public \
+--size 1
+```
+
+The output from the command should be similar to the following:
+```
++--------------------------+--------------------------------------+
+| Field                    | Value                                |
++--------------------------+--------------------------------------+
+| allowed_cidrs            | []                                   |
+| created                  | 2025-02-04T13:08:51                  |
+| datastore                | postgresql                           |
+| datastore_version        | 17.2                                 |
+| datastore_version_number | 17.2                                 |
+| flavor                   | d4a2cb9c-99da-4e0f-82d7-3313cca2b2c2 |
+| id                       | 2f347948-9460-4ac0-a588-32187c8b6ab1 |
+| name                     | pipeline_db_instance                 |
+| operating_status         |                                      |
+| public                   | True                                 |
+| region                   | regionOne                            |
+| service_status_updated   | 2025-02-04T13:08:51                  |
+| status                   | BUILD                                |
+| updated                  | 2025-02-04T13:08:51                  |
+| volume                   | 1                                    |
++--------------------------+--------------------------------------+
+```
+
+In the rest of the tutorial, we assume that:
+
+* the name of the database instance in Pukki is `pipeline_db_instance`
+* the name of the postgresql database is `pipeline_db`
+* the username for the admin of the postgresql database is `db_admin`
+* the password for the admin of the postgresql database is `db_password`
```

Comment:
It is recommended to always encourage users to have complex passwords.

Reply:
after discussion, this should be now fixed in https://github.com/CSCfi/csc-user-guide/pull/2424/commits/9b1b28a56261b814d7262c0afb27932b6881381e